texastribune/styleguides
# Updates to homepage and TribTalk style guide

#### What's the plan?

To update some elements on homepage and TribTalk style guide page for enhanced look and more info.
#### What's this PR do?

Adds in goals/statements copy to homepage for more welcoming info. Adds example code toggles to TribTalk page. Some various styling and spacing adjustments made to homepage and TribTalk style guide as well; see issues linked below under relevant tickets for a more detailed breakdown.
#### How should this be manually tested?

Run styleguides locally and click around to see that everything is looking good.
#### What are the relevant tickets?

This Sprint task: https://basecamp.com/1866004/projects/1175549/todos/119956247
These GitHub issue #10 and #11.
#### Smells?

Don't think so...
#### TODOs:
- [x] Update so that welcome copy isn't included on every page
- [x] Add image to homepage
- [x] Add copy to TribTalk branding section
- [x] Improve TribTalk typography section by fleshing it out a bit more
- [x] Smooth out spacing between sections/headers on TribTalk
#### Technical debt note

Adds more code.
